arduino/smartcar/smartcar.ino

```diff
@@ -103,11 +101,11 @@ void takeInput(String input) {
                 break;
             
               case 4:  //right
-                  car.setAngle(-angle);
+                  car.setAngle(-throttle);
```

Comment:
You are setting the angle to the throttle value? That is very confusing.

Reply:
I know the variables might look confusing that is why I wanted to have two variables with the same input since the android app is giving a incremental number of 10 when you press for example right in the android app.

Reply:
I will change variable to something more understandable i.e "increase"

Reply:
I wouldn't say `increase` is a good name, but it's at least not a confusing one. :P

Reply:
Or actually it is confusing since it implies that the value will result to an increase of some sort which is not necessarily the case. It could very well be a decrease.

Reply:
I renamed variable to appInput now instead hence the speed/angle input is determined by the app.